lib/l10n/app_it.arb

```diff
@@ -0,0 +1,260 @@
+{
+  "guest": "Ospite",
+  "browse": "Esplora",
+  "search": "Cerca",
+  "library": "Libreria",
+  "lyrics": "Testi",
+  "settings": "Impostazioni",
+  "genre_categories_filter": "Filtra categorie o generi...",
+  "genre": "Genere",
+  "personalized": "Personalizzato",
+  "featured": "Featured",
+  "new_releases": "Nuove Release",
+  "songs": "Canzoni",
+  "playing_track": "In riproduzione {track}",
+  "queue_clear_alert": "Questo svuoterà la coda attuale. {track_length} brani saranno rimossi\nVuoi continuare?",
+  "load_more": "Carica altro",
+  "playlists": "Playlist",
+  "artists": "Artisti",
+  "albums": "Album",
+  "tracks": "Brani",
+  "downloads": "Download",
+  "filter_playlists": "Filtra le tue playlist...",
+  "liked_tracks": "Brani Preferiti",
+  "liked_tracks_description": "Tutti i tuoi brani preferiti",
+  "create_playlist": "Crea Playlist",
+  "create_a_playlist": "Crea una playlist",
+  "create": "Crea",
+  "cancel": "Annulla",
+  "playlist_name": "Nome Playlist",
+  "name_of_playlist": "Nome della playlist",
+  "description": "Descrizione",
+  "public": "Pubblico",
+  "collaborative": "Collaborative",
+  "search_local_tracks": "Cerca tra i brani locali...",
+  "play": "Riproduci",
+  "delete": "Elimina",
+  "none": "Nessuno",
+  "sort_a_z": "Ordina da A-Z",
+  "sort_z_a": "Ordina da Z-A",
+  "sort_artist": "Ordina per Artista",
+  "sort_album": "Ordina per Album",
+  "sort_tracks": "Ordina Brani",
+  "currently_downloading": "Attualmente in Downloading ({tracks_length})",
+  "cancel_all": "Annulla Tutto",
+  "filter_artist": "Filtra artisti...",
+  "followers": "{followers} Follower",
+  "add_artist_to_blacklist": "Aggiungi artista a blacklist",
+  "top_tracks": "Migliori Brani",
+  "fans_also_like": "Ai Fans piace anche",
+  "loading": "Caricamento...",
+  "artist": "Artista",
+  "blacklisted": "Blacklisted",
+  "following": "Following",
+  "follow": "Follow",
+  "artist_url_copied": "URL dell'Artista copiato nella clipboard",
+  "added_to_queue": "Aggiunto {tracks} alla coda",
+  "filter_albums": "Filtra album...",
+  "synced": "Synced",
+  "plain": "Plain",
+  "shuffle": "Shuffle",
+  "search_tracks": "Cerca brani...",
+  "released": "Released",
```

Comment:
I think it depends. I don't know what "released" refers to.
It could be "Pubblicato" if used in a sentence like: "this album was released in 2022".

Reply:
We can find it here:

https://github.com/KRTirtho/spotube/blob/6f0f63c286bee5dfe978523a7c047f18dda4b7e8/lib/hooks/use_update_checker.dart#L77

And here:

https://github.com/KRTirtho/spotube/blob/6f0f63c286bee5dfe978523a7c047f18dda4b7e8/lib/components/shared/dialogs/track_details_dialog.dart#L45

So I think it can be translated as "rilasciato" in every caso

Reply:
> So I think it can be translated as "rilasciato" in every caso

Yep, "Rilasciato" sounds better in that context 👍🏻

Reply:
Solved in 8c86486